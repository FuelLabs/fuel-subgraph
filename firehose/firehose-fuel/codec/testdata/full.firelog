FIRE BLOCK_BEGIN 6
FIRE BEGIN_TRX 8aa57a7aecd32837f2b0ea5e3e7713559df949c19220870e9432c1546b9c1ba1 transfer 0xDEADBEEF 0xBAAAAAAD 0 2710 true
FIRE TRX_BEGIN_EVENT 8aa57a7aecd32837f2b0ea5e3e7713559df949c19220870e9432c1546b9c1ba1 token_transfer
FIRE TRX_EVENT_ATTR 8aa57a7aecd32837f2b0ea5e3e7713559df949c19220870e9432c1546b9c1ba1 0 foo bar
FIRE BEGIN_TRX c19f98d87a551217e84c7ab02e6579c1e75f4cffbbbd55f52f8ef6e0164b8b71 transfer 0xDEADBEEF 0xBAAAAAAD 3b9aca00 2710 true
FIRE TRX_BEGIN_EVENT c19f98d87a551217e84c7ab02e6579c1e75f4cffbbbd55f52f8ef6e0164b8b71 token_transfer
FIRE TRX_EVENT_ATTR c19f98d87a551217e84c7ab02e6579c1e75f4cffbbbd55f52f8ef6e0164b8b71 0 foo bar
FIRE BEGIN_TRX 3254289f54b12adc378f2ec18db51254bfa28931ac3c8655769537701744944a transfer 0xDEADBEEF 0xBAAAAAAD 77359400 2710 true
FIRE TRX_BEGIN_EVENT 3254289f54b12adc378f2ec18db51254bfa28931ac3c8655769537701744944a token_transfer
FIRE TRX_EVENT_ATTR 3254289f54b12adc378f2ec18db51254bfa28931ac3c8655769537701744944a 0 foo bar
FIRE BEGIN_TRX 90cddac96f703d56f8a523a6e10230d0e1a9cc6af42c7fe87f8d785976047164 transfer 0xDEADBEEF 0xBAAAAAAD b2d05e00 2710 true
FIRE TRX_BEGIN_EVENT 90cddac96f703d56f8a523a6e10230d0e1a9cc6af42c7fe87f8d785976047164 token_transfer
FIRE TRX_EVENT_ATTR 90cddac96f703d56f8a523a6e10230d0e1a9cc6af42c7fe87f8d785976047164 0 foo bar
FIRE BEGIN_TRX 13bfb5f01b6e4ef41527bc951655e80c7b57d08e3ff98c1ff34483b555e379c8 transfer 0xDEADBEEF 0xBAAAAAAD ee6b2800 2710 true
FIRE TRX_BEGIN_EVENT 13bfb5f01b6e4ef41527bc951655e80c7b57d08e3ff98c1ff34483b555e379c8 token_transfer
FIRE TRX_EVENT_ATTR 13bfb5f01b6e4ef41527bc951655e80c7b57d08e3ff98c1ff34483b555e379c8 0 foo bar
FIRE BEGIN_TRX 49486453e3c4ce9b3a784140041390e3cf7963cb5186b01047ee2bed241920ab transfer 0xDEADBEEF 0xBAAAAAAD 012a05f200 2710 true
FIRE TRX_BEGIN_EVENT 49486453e3c4ce9b3a784140041390e3cf7963cb5186b01047ee2bed241920ab token_transfer
FIRE TRX_EVENT_ATTR 49486453e3c4ce9b3a784140041390e3cf7963cb5186b01047ee2bed241920ab 0 foo bar
FIRE BLOCK_END 6 e7f6c011776e8db7cd330b54174fd76f7d0216b612387a5ffcfb81e6f0919683 ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d 1647087903319636000 6