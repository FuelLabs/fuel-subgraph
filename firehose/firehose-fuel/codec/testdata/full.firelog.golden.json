[{
  "height": 6,
  "hash": "e7f6c011776e8db7cd330b54174fd76f7d0216b612387a5ffcfb81e6f0919683",
  "prevHash": "ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d",
  "timestamp": 1647087903319636000,
  "transactions": [
    {
      "type": "transfer",
      "hash": "8aa57a7aecd32837f2b0ea5e3e7713559df949c19220870e9432c1546b9c1ba1",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {},
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    },
    {
      "type": "transfer",
      "hash": "c19f98d87a551217e84c7ab02e6579c1e75f4cffbbbd55f52f8ef6e0164b8b71",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "O5rKAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    },
    {
      "type": "transfer",
      "hash": "3254289f54b12adc378f2ec18db51254bfa28931ac3c8655769537701744944a",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "dzWUAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    },
    {
      "type": "transfer",
      "hash": "90cddac96f703d56f8a523a6e10230d0e1a9cc6af42c7fe87f8d785976047164",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "stBeAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    },
    {
      "type": "transfer",
      "hash": "13bfb5f01b6e4ef41527bc951655e80c7b57d08e3ff98c1ff34483b555e379c8",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "7msoAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    },
    {
      "type": "transfer",
      "hash": "49486453e3c4ce9b3a784140041390e3cf7963cb5186b01047ee2bed241920ab",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "ASoF8gA="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true,
      "events": [
        {
          "type": "token_transfer",
          "attributes": [
            {
              "key": "foo",
              "value": "bar"
            }
          ]
        }
      ]
    }
  ]
}
]